packages/app-project/.storybook/preview.js

```diff
@@ -0,0 +1,13 @@
+import { addParameters } from '@storybook/react'
+import { INITIAL_VIEWPORTS } from '@storybook/addon-viewport'
+
+import { backgrounds } from './lib'
+
+addParameters({
+  backgrounds: {
+    values: backgrounds.lightDefault
```

Comment:
The dark theme stories need updating to use the new `backgrounds` object structure too.

Reply:
New backgrounds syntax: https://storybook.js.org/docs/react/essentials/backgrounds

I think this should be set in `lib/backgrounds` rather than in the preview config.